report_builder/api/views.py

```diff
@@ -147,10 +147,10 @@ def post(self, request):
                         'field_verbose': field,
                         'field_type': 'Property',
                         'field_choices': None,
-                        'can_filter': True if filters is None or new_field.name in filters else False,
+                        'can_filter': True if filters is None or field in filters else False,
```

Comment:
Oh, I see. Properties and custom_fields are a little different in that way.

If it's easy, could you add or improve a test that would've caught this?